src/client/components/SignInButton/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Button.css';
+import { action } from '@storybook/addon-actions';
+
+const Button = () => {
+  return (
+    <button type="submit" className="button" onClick={action('clicked')}>
```

Comment:
This is wrong place for Storybook action - you only use it in storybook files, like you did below.
Here, you need to pass `onClick` from props.